### DISCUSSION TITLE
Reporting on usage stats from chat history data

### DISCUSSION THREAD

**Discussion by snozwoz** [open-webui/open-webui#7625] [General]
How can i see usage stats by teams and by users?
There must be database with chat history etc, so assume some querys and graphs could be applied to this - would be awesome. 
I found a usage app in the filters, but only for OpenAI, which we are not using, and was thinking is this required given there is chat history DB - not that I know how to connect to it.

**Comment by arunbugkiller** [comment]
Did you find anything on this ? 

If you are running using docker, inside the container you can see a database openwebui.db. You can manipulate that and get the insights from there, maybe explore that and let us know if it works out for you

**Reply by snozwoz** [reply]
hey - thanks for replying. Not found solution - did wonder if I could simply add a service like posthog.com
